pom.xml

```diff
@@ -175,8 +179,8 @@
                     <artifactId>maven-compiler-plugin</artifactId>
                     <version>${maven.compiler.plugin.version}</version>
                     <configuration>
-                        <source>${java.version}</source>
-                        <target>${java.version}</target>
+                        <source>${maven.compiler.source}</source>
```

Comment:
I think that this configuration section is not necessary because the maven compiler plugin autoconfigure itself reading properties `maven.compiler.source` and  `maven.compiler.release`.

-- Nicolas